### ISSUE TITLE
[Bug]: Installation of `@carbon/charts-svelte` fails at postinstall step

### ISSUE THREAD

**Issue by metonym** [carbon-design-system/carbon-charts#1620]
### Name

Eric

### Are you an IBM employee?

- [ ] Yes

### What happened?

I repeatedly experience the following error, even after cleaning `node_modules` and rebuilding `yarn.lock` from scratch.

```sh
Exit code: 127
Command: carbon-telemetry collect --install
Arguments: 
Directory: /<redacted>/carbon-charts-svelte-examples/rollup/node_modules/@carbon/charts-svelte
Output:
/bin/sh: carbon-telemetry: command not found
```

My package.json is as follows:

```json
{
  "private": true,
  "type": "module",
  "scripts": {
    "dev": "vite dev",
    "build": "vite build",
    "preview": "vite preview",
    "check": "svelte-check --tsconfig ./tsconfig.json",
    "check:watch": "svelte-check --tsconfig ./tsconfig.json --watch"
  },
  "devDependencies": {
    "@carbon/charts": "^1.11.9",
    "@carbon/charts-svelte": "^1.11.9",
    "@carbon/styles": "^1.33.1",
    "@sveltejs/adapter-node": "^1.3.1",
    "@sveltejs/kit": "^1.22.3",
    "d3": "^7.8.5",
    "svelte": "^4.0.5",
    "svelte-check": "^3.4.6",
    "typescript": "^5.1.6",
    "vite": "^4.4.3"
  }
}

```

### Version

"@carbon/charts": "^1.11.9"
"@carbon/charts-svelte": "^1.11.9"

### What priority level would this be in your opinion?

P0

**Comment by nstuyvesant** [comment]
Just pushed a commit to[ this PR](https://github.com/carbon-design-system/carbon-charts/pull/1615) for @carbon/charts-svelte

**Comment by nstuyvesant** [comment]
telemetry needed to be a dependency - not devDependency.

**Comment by metonym** [comment]
This was fixed in #1615.

Confirmed it's working now in version 1.11.10.
